pages/layout.js

```diff
@@ -10,8 +15,8 @@ export default function Layout(props) {
   return (
     <div>
       <Head>
-        <title>AI Safety Papers</title>
-        <link rel="icon" href="/papers.png" />
+        <title>{appName}</title>
```

Comment:
I haven't really seen this done like this, but I guess this could be a good stop-gap for other systems later on. Typically when text is pulled apart, it's done in a unique text file (for many pages), in part to help later on with internationalization.